packages/documentation/copy/en/handbook-v2/Type Manipulation/Conditional Types.md

```diff
@@ -205,15 +205,20 @@ type Bools = GetReturnType<(a: boolean, b: boolean) => boolean[]>;
 //   ^?
 ```
 
-When inferring from a type with multiple call signatures (such as the type of an overloaded function), inferences are made from the _last_ signature (which, presumably, is the most permissive catch-all case). It is not possible to perform overload resolution based on a list of argument types.
+When inferring the return type from a type with multiple call signatures, such as an overloaded function, it's essential to note that inferences are made from the _last overload signature_, not from the [implementation signature](https://www.typescriptlang.org/docs/handbook/2/functions.html#overload-signatures-and-the-implementation-signature).
```

Comment:
```suggestion
When inferring the return type from a type with multiple call signatures, such as an overloaded function, inferences are made from the _last overload signature_, not from the [implementation signature](https://www.typescriptlang.org/docs/handbook/2/functions.html#overload-signatures-and-the-implementation-signature).
```

A nit, but I don't think the "it's essential to note" is needed; the entire doc is essential!

Reply:
Thanks for the suggestion! I've removed the "it's essential to note" part. Please review the updated text.